src/cli/commands/version.js

```diff
@@ -124,6 +124,8 @@ export async function setVersion(
       // add manifest
       await spawn('git', ['add', pkgLoc]);
 
+      await config.executeLifecycleScript('version');
```

Comment:
I accidentally formatted the code inside Intellij IDEA and it started getting errors in ESLint.

Reverted the formatting. Everything seems fine now. @Daniel15